PatternPal/PatternPal.Tests/Checks/ParameterCheckTests.cs

```diff
@@ -135,26 +123,24 @@ public Task Parameter_Check_Method_Same_Type()
         IMethod intNode = EntityNodeUtils.GetMemberFromGraph<IMethod>(graph,
             "IntTest",
             "IntTestFunction");
+        
+        TestCheck stringNodeCheck = new(ctx.Graph.Relations.GetEntityByName(stringNode.GetReturnType()));
 
         // TypeCheck of the StringTestFunction method (return type is StringTest)
         TypeCheck typeStringNode = new TypeCheck(
```

Comment:
``
```suggestion
        TypeCheck typeStringNode = new(
```
remove redundant type specfication